.github/styles/Vocab/Base/accept.txt

```diff
@@ -142,39 +136,18 @@ uncheck
 uncomment
 Unicode
 unix
-upstream
-upstreams
+upstreams?
 uri
-url
-urls
+urls?
 uuid
 Valero
-validator
-validators
+validators?
+VMs?
 Vuejs
```

Comment:
I didn't want to change it without updating docs, but I believe we should allow `Vue.js` as it's the correct project name, and disallow `Vuejs`.